src/main/java/me/earth/earthhack/impl/modules/render/popchams/ListenerTotemPop.java

```diff
@@ -26,13 +27,21 @@ public void invoke(TotemPopEvent event) {
             @Override public boolean isCreative() {
                 return false;
             }
+
+            @Override public int getItemInUseCount() {
+                return useCount;
+            }
         };
 
         if (module.copyAnimations.getValue()) {
             playerCopy.setSneaking(player.isSneaking());
+            playerCopy.swingProgress = player.swingProgress;
             playerCopy.limbSwing = player.limbSwing;
             playerCopy.limbSwingAmount = player.prevLimbSwingAmount;
+            playerCopy.inventory.copyInventory(player.inventory);
         }
+        playerCopy.setPrimaryHand(player.getPrimaryHand());
+        playerCopy.ticksExisted = player.ticksExisted;
```

Comment:
More of a question, you know skins can have big arms or small arms? Can we take that into account, too?

Reply:
alright